index.html

```diff
@@ -18,5 +18,157 @@
+            vulputate nisi. Mauris bibendum ante nisl, ac sagittis metus lobortis id. 
+            
+            <div class="read_more">
+              <a href="https://www.gardenersworld.com/offers/travel-and-events/bbc-gardeners-world-spring-fair/">
+              <button>Read More</button></a>
+            </div>
+          </article>
+
+          <hr>
+          
+          <article class="blog">
+            <h3>Second Month of Spring</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sed fermentum tortor, id accumsan velit. Nulla molestie, eros 
+              et porttitor iaculis, odio eros laoreet urna, sed eleifend leo orci eu metus. Nam quis fringilla nibh, ac consequat ex. Ut et pharetra eros, 
+              eu varius turpis. Mauris non placerat turpis, vel aliquet tellus. Mauris ullamcorper lectus in erat auctor varius. Cras auctor nisi quis est
+              eleifend venenatis. Morbi vitae pharetra ipsum. Pellentesque facilisis scelerisque nunc, eget ultricies quam tristique vel. Nulla facilisi. 
+              Duis commodo erat risus, vitae eleifend nisl blandit quis. Cras venenatis risus orci, nec hendrerit turpis porta ac. Sed risus nisi, gravida
+              non pellentesque quis, tincidunt in lorem. Aliquam commodo libero vitae nisl pharetra, quis egestas ipsum euismod. Integer vel fringilla sem. 
+              Maecenas pretium scelerisque tellus non vehicula.Sed et auctor libero.</p>
+            <p>Integer gravida sem et lobortis suscipit. Vivamus fringilla nisl vel neque rhoncus sodales. Suspendisse potenti. Vestibulum 
+              ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras ex nulla, condimentum at sagittis non, consectetur 
+              blandit ligula. Curabitur semper neque eget odio malesuada volutpat. Vivamus gravida imperdiet metus, tempor volutpat est venenatis nec. 
+              Curabitur egestas euismod fringilla. Nulla volutpat urna in odio finibus placerat. Aenean viverra ultrices bibendum. Duis non efficitur odio.</p>
+            <p>sit amet fermentum est iaculis sed. Vivamus dolor dolor, semper et mattis et, porttitor eu turpis. Quisque sed pharetra ligula. 
+              Vivamus elementum, nisi sit amet gravida pulvinar, orci libero dignissim eros, ac efficitur arcu dui id nibh. Cras sit amet vestibulum orci. 
+              Ut sed tempor arcu. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nulla nec sapien justo. In ac 
+              vulputate nisi. Mauris bibendum ante nisl, ac sagittis metus lobortis id. 
+              
+              <div class="read_more">
+              <a href="https://www.gardenersworld.com/offers/travel-and-events/bbc-gardeners-world-spring-fair/">
+              <button>Read More</button></a>
+              </div>
```

Comment:
The tags here are not correctly indented - usually you will want a child element to be indented 1 level (e.g. a tab or 2 spaces) from its parent. There are a few places where this is the case - remember that you can format your HTML within VS Code;

On Windows Shift + Alt + F.
On Mac Shift + Option + F.
On Linux Ctrl + Shift + I.

Reply:
Hi Paul, I followed your suggestion and used an auto formatter in VS code. Thank you.